src/components/TextField/TextField.tsx

```diff
@@ -23,13 +28,38 @@ export const TextField: React.FC<Props> = ({
   placeholder = `Enter ${label}`,
   required = false,
   onChange = () => {},
+  count,
+  setCount,
+  isValidImgUrl,
+  isValidImdbUrl,
 }) => {
-  // generage a unique id once on component load
   const [id] = useState(() => `${name}-${getRandomDigits()}`);
 
-  // To show errors only if the field was touched (onBlur)
   const [touched, setTouched] = useState(false);
+
+  const checkCount = () => {
+    if (count > 0) {
+      setTouched(false);
+      setCount(0);
+    }
+  };
+
+  useEffect(() => {
+    checkCount();
+  }, [count]);
+
   const hasError = touched && required && !value;
```

Comment:
```suggestion
  const hasError = touched && required && !value.trim();
```
found one more bug.
now user can input all spaces in some fields, no visible any errors after that, but button still disabled.
suggestion can help to fis this issue
![Screenshot from 2023-09-04 17-46-46](https://github.com/mate-academy/react_movies-list-add-form/assets/33029864/f5ffd1cc-9a61-4014-b4d2-f323f03a60f1)